app/main.py

```diff
@@ -1,8 +1,31 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for person in people:
+        instance = Person(person["name"], person["age"])
+        person_list.append(instance)
+        if "husband" in person and person["husband"] is not None:
+            instance.husband = person["husband"]
+        if "wife" in person and person["wife"] is not None:
+            instance.wife = person["wife"]
```

Comment:
There is no need to assign name to a `husband` or `wife` attribute that will be reassigned later with an instance. Just create all `Person` class instances as a first step to populate `Person.people`